src/styles/_colors.scss

```diff
@@ -16,6 +18,7 @@ $tertiary-color: #E9D7A1 !default; /* compliments the primary color, usually use
 
 $alt-color: #786452 !default; /* for all other uses */
 $alt-dark: #443730 !default;
+$gold-brown: #956d58 !default;
```

Comment:
This color is used by mdizo several times, and I'm also using it, so I figured it should probably be a variable